packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -205,7 +202,7 @@ export default {
          */
         checkCookieBannerCookie () {
             if (this.legacyBanner) {
-                this.shouldHideBanner = CookieHelper.get(this.legacyConsentCookieName) === 2;
+                this.shouldHideBanner = CookieHelper.get(this.legacyConsentCookieName) === '130315';
```

Comment:
What does "130315" refer to?

Reply:
@DamianMullins I think it refers to a cookie policy version number:
![image](https://user-images.githubusercontent.com/1398088/109151275-dab8a880-7761-11eb-9159-a7368cbc8955.png)

Reply:
But this is for the legacy cookie banner functionality, which we are replacing, so I hard-coded it cos we'll be deleting it soon anyway

Reply:
Thanks @simonsandersje